### ISSUE TITLE
Feature request: build pull request

### ISSUE THREAD

**Issue by hannesa2** [gradle-update/update-gradle-wrapper-action#108]
I use this action on several repos, but compared to [dependabot pull request](https://github.com/MikeOrtiz/TouchImageView/pull/310), the pull requests which are created with this action are [not build on CI](https://github.com/MikeOrtiz/TouchImageView/pull/311)

Any chance to change this and let them build out of the box ?

**Comment by huehnerlady** [comment]
@hannesa2 I got a similar issue where the push event is not triggered as there the github token is used which does not trigger events (#83)

I can see though that you are having the pull request  trigger as well. But currenly you are using the GITHUB_TOKEN for the build. This does not trigger any events and therefore the CI builds can't be triggered (I learned this in a discussion with GitHub directly). You need to change this line:
`          repo-token: ${{ secrets.GITHUB_TOKEN }}` in your wrapper 
 
 Basically you need to add a PAT for that (we use a PAT from a technical github user).
 It is not ideal and I asked GitHub to add a flag where you can say that you want to have events triggered with GITHUB_TOKEN, but I do not think we will have that anytime soon, if at all.
 So if you change your token to a PAT it should trigger the build :)

**Comment by hannesa2** [comment]
> So if you change your token to a PAT it should trigger the build :)

Sadly, I'm only the maintainer and I can't go into settings tab to do anything with a token. The owner does nothing

How does the dependabot works ? They don't need a token at all

**Comment by huehnerlady** [comment]
> > So if you change your token to a PAT it should trigger the build :)
> 
> Sadly, I'm only the maintainer and I can't go into settings tab to do anything with a token. The owner does nothing
> 
> How does the dependabot works ? They don't need a token at all

I am not sure but I assume as dependabot is part of the github infrastructure it is doing something different, but I cannot tell you the details I'm afraid

**Comment by cristiangreco** [comment]
@hannesa2 sorry for the late reply, and @huehnerlady thanks for jumping in!

I'm putting together an FAQ entry, does this text make sense to you? 👉 https://github.com/gradle-update/update-gradle-wrapper-action/tree/repo-token-default#running-ci-workflows-in-pull-requests-created-by-the-action

As I understand, you don't have admin access to the repository so the only option you've got in order to trigger build and/or CI checks is to manually close and re-open the PR.

**Comment by hannesa2** [comment]
> As I understand, you don't have admin access to the repository so the only option you've got in order to trigger build and/or CI checks is to manually close and re-open the PR.

exactly !
I've to change PR, then it builds
